src/api/apiutil/artemis_jolokia.ts

```diff
@@ -648,21 +303,53 @@ export class ArtemisJolokia {
     return JSON.stringify([{ type: 'read', mbean: bean }]);
   };
 
+  getComponentNameKey(compType: ComponentType): string {
+    switch (compType) {
+      case ComponentType.ACCEPTOR: {
+        return 'ACCEPTOR_NAME';
+      }
+      case ComponentType.ADDRESS: {
+        return 'ADDRESS_NAME';
+      }
+      case ComponentType.BROKER: {
+        return '';
```

Comment:
Hello @gaohoward ,
here it should return 'BROKER_NAME' right?? instead of ' ' string...

Reply:
Hi Marjina, this is a special case for `BROKER` component. We use this method to find the key to replace component names. The `BROKER` component is not used here as we always search for it to replace for each component (each component query will have `BROKER_NAME` search and replace. Here right before calling this method:
https://github.com/gaohoward/activemq-artemis-jolokia-api-server/blob/j_qaddr_ops/src/api/apiutil/artemis_jolokia.ts#L341